etc/lf.nu

```diff
@@ -2,24 +2,27 @@
 # 
 # Documentation: https://www.nushell.sh/book/externs.html
 
-# Either add this to your existing completions or copy this to
-# your nu config file (Use 'config nu' in the nushell to open it).
+# You may put this file into the :
```

Comment:
Just one more minor comment, I think you missed some text here.

Reply:
That's OK, I have never used `nushell` before so it was a learning experience for me as well 🙂.

Thanks very much for the changes once again.